VideoLocker/src/androidTest/java/org/edx/mobile/test/NotificationTest.java

```diff
@@ -0,0 +1,118 @@
+package org.edx.mobile.test;
+
+import org.edx.mobile.model.api.CourseEntry;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.module.notification.EdxLocalParseChannel;
+import org.edx.mobile.module.notification.NotificationDelegate;
+import org.edx.mobile.module.notification.NotificationPreference;
+import org.edx.mobile.module.notification.UserNotificationManager;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NotificationTest extends BaseTestCase {
+
+    private static NotificationDelegate tracker;
+    private static UserNotificationManager manager;
+
+    private final String  courseId1 = "course_id_1";
+    private final String  courseId2 = "course_id_2";
+    private final String  courseId3 = "course_id_3";
+    private final String  channelId1 = "channel_id_1";
+    private final String  channelId2 = "channel_id_2";
+    private final String  channelId3 = "channel_id_3";
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        initDelegation();
+    }
+
+    private void initDelegation() {
+
+        if (manager == null) {
+            tracker = Mockito.mock(NotificationDelegate.class);
+
+            manager =   UserNotificationManager.getInstance();
```

Comment:
Can we just instantiate one of these in the test instead of using a singleton instance?